app/controllers/projects/notes_controller.rb

```diff
@@ -61,7 +66,7 @@ def delete_attachment
   private
 
   def note
-    @note ||= @project.notes.find(params[:id])
+    @note ||= Note.find(params[:id])
```

Comment:
Ah yes, I think this would now technically allow one to specify _any_ note ID in the "id" parameter and get it to render the note, thus we should keep the `@project.notes.find` form.